docs/site/content/en/docs/Tasks/experiment-evaluation/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Dataset Experiment Evaluation
+description: Document explaining the use of Experiment Evaluation function in Horreum UI
+date: 2024-01-18
+weight: 5
+---
+
+## Using the Dataset Experiment Evaluation View
+
+Using the Experiment evaluation window you can quickly see the values and the relative difference of a Run.
+Start by initially loading the Test. Then click the `Dataset list` button.
+
+{{% imgproc dataset_list Fit "1200x900" %}} Dataset List {{% /imgproc %}}
+
+Then select the individual Test Run you want to compare with it's Baseline.
+
+By navigating to the uploaded run Dataset view page you will see a button  “Evaluate experiment”. Clicking this button opens a window revealing the comparison result for the current Run and the Baseline Run..
+
+{{% imgproc individual_evaluation Fit "1200x900" %}} Individual Evaluation {{% /imgproc %}}
+
+Results show the values then the percentage difference.
+
+
+Datasets Comparison View
+
+Horreum provides multiple Run comparisons in the Dataset Comparison View. We can filter based on the Filter labels defined in the Schema.
+
+Start by initially loading the Dataset list. Then click the button “Select for comparison”.
+
+{{% imgproc select_for_comparison Fit "1200x900" %}} Select for comparison {{% /imgproc %}}
+
+Next the Comparison view is displayed. This is where filters are set. Resize horizontally the window to reveal additional Filter options if obscured.
```

Comment:
`Resize horizontally the window to reveal additional Filter options if obscured.` Is this a UI issue? 

If it is, I think we should log it as an issue and omit this sentence

Reply:
Yes this issue is already created. I added it here to suggest a workaround.
Will remove this.